static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -14,14 +14,8 @@ import { isInVariantSynchronous } from 'common/modules/experiments/ab';
 import { googletagPrebidEnforcement } from 'common/modules/experiments/tests/tcfv2-googletag-prebid-enforcement';
 
 const SOURCEPOINT_ID: string = '5f22bfd82a6b6c1afd1181a9';
-let moduleLoadResult = Promise.resolve();
-
-if (!isGoogleProxy()) {
-    moduleLoadResult = import(/* webpackChunkName: "Prebid.js" */ 'prebid.js/build/dist/prebid');
```

Comment:
This line was executed regardless of the specific conditions to run it
```
dfpEnv.hbImpl.prebid &&
            commercialFeatures.dfpAdvertising &&
            !commercialFeatures.adFree &&
            !config.get('page.hasPageSkin') &&
            !isGoogleProxy() &&
            !shouldIncludeOnlyA9
```

Including regardless the result of ` onConsentChange`

Reply:
How can we test this does not break Google proxies?

Reply:
The condition is still there ` !isGoogleProxy()` line 24. It was referenced twice